llvm/include/llvm/Support/ModRef.h

```diff
@@ -232,7 +243,10 @@ template <typename LocationEnum> class MemoryEffectsBase {
 
   /// Whether this function only (at most) accesses inaccessible memory.
   bool onlyAccessesInaccessibleMem() const {
-    return getWithoutLoc(Location::InaccessibleMem).doesNotAccessMemory();
+    return getWithoutLoc(IRMemLocation::TargetMem0)
+        .getWithoutLoc(IRMemLocation::TargetMem1)
```

Comment:
To be consistent `onlyAccessesInaccessibleOrArgMem` also needs the same additions.